build/source/dshare/globalData.f90

```diff
@@ -55,6 +55,10 @@ MODULE globalData
  ! * part 1: parameters that are fixed across multiple instantiations
  ! ----------------------------------------------------------------------------------------------------------------
 
+ ! define version number
+ character(len=256),parameter,public         :: summaVersion='v2.34.56'
+`
```

Comment:
This does not have anything to do with implementing the midpoint method. Please separate the tasks into independent PRs.